.github/scripts/changelog_author.py

```diff
@@ -0,0 +1,12 @@
+import yaml
+import sys
+
+rule_file = sys.argv[1]
+with open(rule_file, 'r') as stream:
```

Comment:
recommend: prefer to use `rb` with an explicit call to `f.read().decode("utf-8")`

no need to change here, just mentioning it

Reply:
can you please elaborate on why?

Reply:
to be clear, this is a personal style thing.

basically, because the behavior here is platform-dependent:

![image](https://user-images.githubusercontent.com/156560/118321571-00309500-b4bb-11eb-81e8-2193348c640a.png)


while it probably doesn't matter in this case (CI is always going to be linux, and files never shared across platforms) doing things explicitly prevents subtle bugs for creeping in when files might be shared. therefore, i default to using explicit encodings in all cases.

Reply:
Thanks for the explanation @williballenthin!